elyra/api/elyra.yaml

```diff
@@ -342,6 +342,35 @@ paths:
                     type: string
                     description: The pipeline export path.
 
+  /elyra/contents/properties/{path}:
+    get:
+        tags:
+        - properties
+        summary: Get a dictionary of properties for a file.
+        parameters:
+        - name: path
+          in: path
+          description: The path to the file to be parsed, relative to the JupyterLab server root directory.
+          required: true
+          schema:
+            type: string
+        responses:
+          200:
+            description: The dictionary of properties found while parsing the file.
+            content:
+              application/json:
+                schema:
+                  type: object
+                  description: A dictionary of property dictionaries. Each property dictionary consists of a key for each instance found and its value (if applicable), or, if parsing fails, is empty.
+                  oneOf:
+                    - $ref: '#/components/schemas/ResourceBody'
```

Comment:
The convention we've been using is that this name reflects the resource relative to the endpoint.  For example, we should probably use `ContentsPropertiesResource` or `ContentsPropertiesResponse`.

Looks like metadata chose to use _Resource_, while pipeline chose _Response_.  I think _Resource_ is more conformant to the open API spec terminology, but it probably doesn't make a difference.

The _Body_ term applies more to _input_ to POST and PUT calls.

Reply:
Ah that does make more sense, especially considering a GET request wouldn't have a body. `ContentsPropertiesResource` it is!